cmd/kubeapps-apis/docs/kubeapps-apis.swagger.json

```diff
@@ -1826,6 +1901,58 @@
       "description": "Response for GetInstalledPackageSummaries",
       "title": "GetInstalledPackageSummariesResponse"
     },
+    "v1alpha1InstalledPackageDetail": {
+      "type": "object",
+      "properties": {
+        "installedPackageRef": {
+          "$ref": "#/definitions/v1alpha1InstalledPackageReference",
+          "description": "A reference uniquely identifying the package.",
+          "title": "InstalledPackageReference"
+        },
+        "pkgVersionReference": {
+          "$ref": "#/definitions/v1alpha1VersionReference",
+          "description": "The package version reference defines a version or constraint limiting\nmatching package versions.",
+          "title": "PkgVersionReference"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name given to the installed package",
+          "title": "Installed package name"
+        },
+        "currentPkgVersion": {
+          "type": "string",
+          "description": "The version of the package which is currently installed.",
+          "title": "CurrentPkgVersion"
+        },
+        "valuesApplied": {
+          "type": "string",
+          "description": "The values applied currently for the installed application.",
```

Comment:
double checking terminology, is it intended to be "installed application" or "installed package"?